app.js

```diff
@@ -33,17 +21,27 @@ async function getToken({ params }, settings) {
     expires,
     reqTime.utc().format(dateFormat),
   );
-  const { cluster } = await EKS.describeCluster({ name: clusterName }).promise();
-  const expirationTime = reqTime.add(expires, "s").utc().format(dateFormat);
+  const { cluster } = await eks.describeCluster({ name: clusterName }).promise();
+  const expirationTimestamp = reqTime.add(expires, "s").utc().format(dateFormat);
   return {
-    expirationTimestamp: expirationTime,
+    expirationTimestamp,
     token,
     clusterHost: cluster.endpoint,
     clusterCA: cluster.certificateAuthority.data,
   };
 }
 
+async function createCluster({ params }, settings) {
+  const { accessKeyId, secretAccessKey, region } = mapAwsConfig(params, settings);
+  const clusterPayload = createPayloadForCreateCluster(params);
```

Comment:
```suggestion
  const clusterPayload = createPayloadForCreateCluster(params);
  
  const { accessKeyId, secretAccessKey, region } = mapAwsConfig(params, settings);
```
Tiny reordering emphasizing that there are other similar blocks in other functions, which might give a hint for some further refactoring.